typescript/packages/plugins/balancer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@goat-sdk/plugin-balancer",
+    "version": "0.1.0",
+    "main": "./dist/index.js",
+    "module": "./dist/index.mjs",
+    "types": "./dist/index.d.ts",
+    "files": ["dist"],
+    "scripts": {
+        "build": "tsup",
+        "clean": "rm -rf dist",
+        "dev": "tsup src/index.ts --format esm,cjs --watch --dts",
+        "lint": "eslint src/",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@balancer/sdk": "^1.1.0",
+        "@goat-sdk/core": "workspace:*",
+        "@goat-sdk/wallet-evm": "workspace:*",
+        "viem": "^2.7.8",
+        "zod": "catalog:"
+    },
+    "publishConfig": {
```

Comment:
Not needed, use the default package.json from another plugin instead

Reply:
okay done